core/plannernotes.c

```diff
@@ -35,11 +35,16 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 {
 	const unsigned int sz_buffer = 2000000;
 	const unsigned int sz_temp = 100000;
+	const unsigned int sz_icdbuf = 10000;
 	char *buffer = (char *)malloc(sz_buffer);
 	char *temp = (char *)malloc(sz_temp);
+	char *icdbuffer = (char *)malloc(sz_icdbuf);
+	char *prev_gasname = (char *)malloc(10);
+	char *colorc = (char *)malloc(10);
```

Comment:
I would avoid using this variable for the color code...
See comment below.

Reply:
And worse ... any new malloc shall be freed somewhere. This leaks memory, and will definitely trip new Coverity analysis warnings.

Reply:
Yep, you're right.
I guess we could even avoid the variable for prev_gasname.